src/HealthcareApis/HealthcareApis.Test/HealthcareApisService.Test.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>HealthcareApisFhirService</PsModuleName>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.Test.props" />
+
+  <PropertyGroup>
+    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix)</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.HealthcareApis" Version="1.0.1-preview" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.HealthcareApis">
+      <HintPath>..\..\..\..\Azure-SDK\artifacts\bin\Microsoft.Azure.Management.HealthcareApis\Debug\netstandard2.0\Microsoft.Azure.Management.HealthcareApis.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```

Comment:
Remove this `<ItemGroup>` entirely. Replace with:
```xml
  <ItemGroup>
    <ProjectReference Include="..\HealthcareApis\HealthcareApis.csproj" />
  </ItemGroup>
```

Reply:
Done